Source/interfaces/json/DisplayInfo.json

```diff
@@ -71,12 +71,203 @@
           "hdrtype"
         ]
       }
+    },
+    "videoPlaybackResolution": {
```

Comment:
Thunder is designed with consistency in mind. Take note that currently "all" JSON interfaces are designed lower case (tools will automatically make them lowercase) definigin it in the source as camelcase might confuse development. Please comply to the coding guidelines/rules.

Reply:
Done